client/components/social-buttons/google.js

```diff
@@ -134,14 +152,14 @@ class GoogleLoginButton extends Component {
 
 				return Promise.reject( error );
 			} );
-
 		return this.initialized;
 	}
 
 	handleClick( event ) {
 		event.preventDefault();
 
-		if ( this.state.isDisabled ) {
+		if ( ! this.state.isInitilized && ! this.state.isClicked ) {
+			this.setState( { isClicked: event, isLoading: true } );
```

Comment:
`isClicked` should probably be renamed if it's going to hold non-boolean values. I'd rename it to something like `clickEvent` and default it to `undefined` instead of `false`.